Source/Modules/go.cxx

```diff
@@ -1822,15 +1822,15 @@ class GO:public Language {
     // Write out the result type.
     if (is_constructor) {
       String *cl = exportedName(class_name);
-      Printv(parm_print, " (_swig_ret ", cl, ")", NULL);
+      Printv(parm_print, " ", cl, "", NULL);
       if (first == NULL) {
 	first = NewString("_swig_ret");
```

Comment:
Note how if there are no parameters, only results, first will be set to _swig_ret, and then referenced below.  This change can't work in that case.

Reply:
I see. Thanks for the review. This was admittedly a bit of a naive attempt.

Reply:
@tmc Are you planning to revise this patch, or should we just close it?

Reply:
I’ll close it.